src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
-export interface Figure {
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
 
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
 }
 
-export class Triangle {
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
+function isNegative(first: number, second: number, third: number = 1): void {
```

Comment:
function name should contain some action
check what you can do with this function and create corresponding function name

also, don't use separate argument for each variable
if you'll have some figure with 10 sides, will you create the variable for each of them?

let's move all of them to the one array and will process it